_layouts/home.html

```diff
@@ -31,9 +31,11 @@ <h1 style="color: white">
                 <h2 style="color: white">Projects</h2>
                 <p><a class="sm-a" href="mailto:rse@sheffield.ac.uk">Add fractional RSE time from the team to your funding application <br> (email rse@sheffield.ac.uk).</a></p>
             </div>
+
+
             <div class="col-md text-center">
-                <h2 style="color: white">CMI-RSE Projects [Re-open in Nov. 2024]</h2>
-                <p><a class="sm-a" href="/collaboration/cmi-rse">Dedicated AI and ML RSE support in collaboration with the Centre for Machine Intelligence at the University of Sheffield.</a></p>
+                <h2 style="color: white">FAIR<sup>2</sup> Training program</h2>
+                <p><a class="sm-a" href="/training/fair4rs">Dedicated lecures to apply FAIR principles to research softwares and make you research more open.</a></p>
```

Comment:
```suggestion
                <p><a class="sm-a" href="/training/fair4rs">Dedicated lectures to apply FAIR principles to research software and make you research more open.</a></p>
```

Tbh it's not just lectures

Reply:
> Dedicated lectures and training, teaching you how to apply FAIR principles to research software so that you can make your research more open.

I'm not super happy with my attempt at rephrasing.

?